drivers/clock_control/clock_stm32_ll_h5.c

```diff
@@ -434,17 +434,23 @@ static int set_up_plls(void)
 #endif
 
 #if defined(STM32_PLL_ENABLED)
+
+#if defined(CONFIG_STM32_MEMMAP) && defined(CONFIG_BOOTLOADER_MCUBOOT)
```

Comment:
Intentionally removing switching sysclk source to HSI?
If so, I think it should be mentioned in the commit message.
Also if so, can remove `clock_switch_to_hsi()` local helper function.

Reply:
switching the clock to HSI in case the pll1 is not the source clock of the xspi peripheral
--> `#else` part